_posts/en/newsletters/2019-07-24-newsletter.md

```diff
@@ -0,0 +1,149 @@
+wiki page for changes -->{% endcomment %}
+
+{% include specials/bech32/19-real-fees.md %}
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #15891][] changes the node defaults when using
+  regtest mode to enforce the same standard transaction rules used on
+  mainnet.  Those rules determine which transactions nodes relay and
+  accept into the mempool.  This change should make it easier for
+  developers to test their custom transactions against the default
+  policy.  Anyone who needs the old behavior of relaying and accepting
+  any consensus-valid transaction may use the `acceptnonstdtxn`
+  configuration parameter.
+
+- [Bitcoin Core #16152][] disables [BIP37][] bloom filter support by
+  default.  This feature allowed lightweight wallets to create a [bloom
+  filter][] from a list of their addresses, send that filter to a node,
+  ask the node to scan historic blocks or new incoming transactions, and
+  receive back only those transactions that matched the filter.  This
+  allowed the lightweight client to only receive the transactions it was
+  interested in (plus maybe a few extra false-positive matches),
+  reducing its bandwidth requirements.  However, it also meant that
+  clients could require full nodes to scan and rescan through the entire
+  historic block chain over and over at no cost to the client---creating
+  the vector for a DoS attack.
+
+    For that reason and others (including [privacy concerns][filter
+    privacy]) several Bitcoin Core contributors have wanted to disable
+    the feature for several years now.  Early efforts to that end
+    included adding a [BIP111][] services flag to indicate whether or
+    not a node supports bloom filters so that clients can find
+    supporting nodes, and a `peerbloomfilters` configuration option that
+    allows node users to disable bloom filters in case they're worried
+    about the DoS attack.  Additionally, bloom filter support was never
+    updated for checking the contents of the new witness field after
+    segwit was activated, making it less useful than it could be for
+    segwit wallets.
+
+    With this PR #16152, the bloom filter configuration option is now
+    set by default to off.  Users who still want to serve bloom filters
+    can re-enable it.  More notably, many nodes continue to run old versions
+    for years after newer versions have become available, so it's
+    expected that developers of wallets using bloom filters will have
+    some time after the release of Bitcoin Core 0.19
+    ([estimated][Bitcoin Core #15940] late 2019) to find a replacement
+    source of data.
+
+- [Bitcoin Core #15681][] adds an exception to the Bitcoin Core's
```

Comment:
Possibly either remove "the" or s/Core's/Core/

Reply:
removed "the"